src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: square, area: " + calculateArea() + " sq. units, side: "
+                + getSide() + " units, color: " + getColor());
```

Comment:
Use only English in messages/code. Avoid using abbreviations like 'sq.' for 'square' to maintain clarity and professionalism in the code.